components/event-handler-notification/org.wso2.carbon.identity.event.handler.notification/src/main/java/org/wso2/carbon/identity/event/handler/notification/util/NotificationUtil.java

```diff
@@ -448,12 +448,16 @@ public static String getBrandingPreference(String key, JsonNode brandingPreferen
                 break;
             case "organization.color.primary" :
                 value = brandingIsEnabled
-                        ? getBrandingPreferenceByTheme(brandingPreferences, theme, "/colors/primary")
+                        ? !StringUtils.isBlank(getBrandingPreferenceByTheme(brandingPreferences, theme, "/colors/primary/main"))
```

Comment:
Can we use [`StringUtils.defaultIfBlank`](https://commons.apache.org/proper/commons-lang/apidocs/org/apache/commons/lang3/StringUtils.html#defaultIfBlank-T-T-) here to avoid nested ternary expressions ?

Reply:
+1 That'll be better.

Reply:
I'll send a follow up PR for this .